x-pack/test/api_integration/apis/cloud_security_posture/update_rules_config.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import Chance from 'chance';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+  const chance = new Chance();
+
+  describe('POST /internal/cloud_security_posture/update_rules_config', () => {
+    let agentPolicyId: string;
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/empty_kibana');
+      await esArchiver.load('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/empty_kibana');
+      await esArchiver.unload('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+    });
+
+    before(async function () {
+      const { body: agentPolicyResponse } = await supertest
+        .post(`/api/fleet/agent_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'Test policy',
+          namespace: 'default',
+        });
+      agentPolicyId = agentPolicyResponse.item.id;
+    });
+
+    it(`should return 500 when package policy id not exist`, async () => {
+      const packagePolicyId = chance.guid();
+
+      const { body: response } = await supertest
+        .post(`/internal/cloud_security_posture/update_rules_config`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          package_policy_id: packagePolicyId,
+        })
+        .expect(500);
+      expect(response.error).to.be('Internal Server Error');
+      expect(response.message).to.be(`package policy Id '${packagePolicyId}' is not exist`);
+    });
+
+    it(`should return 200 for existing package policy id`, async () => {
+      const { body: postPackageResponse } = await supertest
+        .post(`/api/fleet/package_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          force: true,
+          name: 'cloud_security_posture-1',
+          description: '',
+          namespace: 'default',
+          policy_id: agentPolicyId,
+          enabled: true,
+          output_id: '',
+          inputs: [],
+          package: {
+            name: 'cloud_security_posture',
+            title: 'Kubernetes Security Posture Management',
+            version: '0.0.26', // TODO: Find a method of maintaining the most recent version of the package
```

Comment:
It's seems that [Fleet API](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/elastic/kibana/master/x-pack/plugins/fleet/common/openapi/bundled.json#/) has package info API only by version

Reply:
@joshdover, is there a way to get the latest package version by package name?